core/src/wallet/bitcoin/networks.hpp

```diff
@@ -45,6 +45,15 @@ namespace ledger {
     namespace core {
 
         namespace networks {
+
+            enum sigHashType : uint8_t {
+                SIGHASH_ALL = 1,
+                SIGHASH_NONE = 2,
+                SIGHASH_SINGLE = 3,
+                SIGHASH_FORKID = 0x40,
```

Comment:
Can you change the way we write those literals? Mixing decimals with hexadecimals is _wat_.